txscript/engine.go

```diff
@@ -198,6 +205,8 @@ func (vm *Engine) curPC() (script int, off int, err error) {
 	return vm.scriptIdx, vm.scriptOff, nil
 }
 
+//func (vm *Engine) verifyWitnessProgram(uint version, wi
```

Comment:
This appears to be left over.  I suspect the intention was to factor the code out of `Step` which I think makes sense.